docs/getting-started-monitoring.md

```diff
@@ -9,15 +9,18 @@ We will create the following environment in your local by using Docker and minik
 ```
 +--------------------------------------------------------------------------------------------------+
 | +------------------------------------------------------+                     +-----------------+ |
-| |                kube-prometheus-stack                 |                     | Scalar Products | |
-| |                                                      |                     |                 | |
-| | +--------------+  +--------------+  +--------------+ | ---(Monitoring)---> |  +-----------+  | |
-| | |  Prometheus  |  | Alertmanager |  |   Grafana    | |                     |  | Scalar DB |  | |
-| | +-------+------+  +------+-------+  +------+-------+ |                     |  +-----------+  | |
-| |         |                |                 |         |                     |  +-----------+  | |
-| |         +----------------+-----------------+         |                     |  | Scalar DL |  | |
-| |                          |                           |                     |  +-----------+  | |
-| +--------------------------+---------------------------+                     +-----------------+ |
+| |                kube-prometheus-stack                 | ---(Monitoring)---> | Scalar Products | |
+| |                                                      |         |           |                 | |
+| |                                                      |  +------+-------+   |                 | |
+| |                                                      |  |  loki-stack  |   |                 | |
+| |                                                      |  |              |   |                 | |
+| | +--------------+  +--------------+  +--------------+ |  | +----------+ |   |  +-----------+  | |
+| | |  Prometheus  |  | Alertmanager |  |   Grafana    | |  | | Promtail | |   |  | Scalar DB |  | |
+| | +-------+------+  +------+-------+  +------+-------+ |  | +----------+ |   |  +-----------+  | |
+| |         |                |                 |         |  | +----------+ |   |  +-----------+  | |
+| |         +----------------+-----------------+         |  | |   Loki   | |   |  | Scalar DL |  | |
+| |                          |                           |  | +----------+ |   |  +-----------+  | |
+| +--------------------------+---------------------------+  +--------------+   +-----------------+ |
 |                            |                                                                     |
 |                            |         Kubernetes (minikube)                                       |
 +----------------------------+---------------------------------------------------------------------+
```

Comment:
I think it is better to clarify each role of `kube-prometheus-stack` and `loki-stack` as follows.
```
+--------------------------------------------------------------------------------------------------+
| +------------------------------------+                                                           |
| |             loki-stack             |                                                           |
| |                                    |                                       +-----------------+ |
| | +--------------+  +--------------+ | ---------------------(Logging)------> | Scalar Products | |
| | |     Loki     |  |   Promtail   | |                                       |                 | |
| | +--------------+  +--------------+ |                                       |  +-----------+  | |
| +------------------------------------+                                       |  | Scalar DB |  | |
|                                                                              |  +-----------+  | |
| +------------------------------------------------------+                     |                 | |
| |                kube-prometheus-stack                 |                     |  +-----------+  | |
| |                                                      |                     |  | Scalar DL |  | |
| | +--------------+  +--------------+  +--------------+ | ---(Monitoring)---> |  +-----------+  | |
| | |  Prometheus  |  | Alertmanager |  |   Grafana    | |                     +-----------------+ |
| | +-------+------+  +------+-------+  +------+-------+ |                                         |
| |         |                |                 |         |                                         |
| |         +----------------+-----------------+         |                                         |
| |                          |                           |                                         |
| +--------------------------+---------------------------+                                         |
|                            |                                                                     |
|                            |         Kubernetes (minikube)                                       |
+----------------------------+---------------------------------------------------------------------+
```

Reply:
Thank you for the advice!
I revised it in https://github.com/scalar-labs/helm-charts/pull/87/commits/91cad7c3414a5b46c76959babae3594c65cad87a